website/docs/dbt-cli/install/docker.md

```diff
@@ -3,8 +3,117 @@ title: "Use Docker to install dbt"
 description: "You can use Docker to install dbt and adapter plugins from the command line."
 ---
 
-By v1.0.0, dbt Core and all adapter plugins maintained by dbt Labs will be available as official Docker images,
-and available from a public registry.
-We recommend you use Docker to install in production because it includes dbt Core and all of its dependencies. You might might also use a Docker to install and develop locally if you don't have your python environment set up.
+dbt Core and all adapter plugins maintained by dbt Labs will are available as [Docker](https://docs.docker.com/) images. We recommend you use Docker to install dbt in production because it includes dbt Core, one or more database adapters, and all of their dependencies. You may also be able to use Docker to install and develop locally if you don't have a python environment set up.
```

Comment:
```suggestion
You will find dbt Core and all adapter plugins maintained by dbt Labs available as [Docker](https://docs.docker.com/) images. We recommend you use Docker to install dbt in production because it includes dbt Core, one or more database adapters, and all dependencies. You can use Docker to install and develop locally if you don't have a python environment set up.
```